### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by tammycamp** [chrisle/seer.js-public#9]
I've made a copy of the spreadsheet and it gets an error every time I input the keyword.

**Comment by ReessKennedy** [comment]
Yeah, me too. I think Google disabled some of the functionality. @chrisle ?
